forms-shared/src/schemas/tsb/objednavkaVytycenie.ts

```diff
@@ -0,0 +1,252 @@
+import {
+  schema,
+  step,
+  radioGroup,
+  input,
+  object,
+  checkbox,
+  conditionalFields,
+  select,
+  datePicker,
+  fileUpload,
+  selectMultiple,
+} from '../../generator/functions'
+import { createStringOptions, createCondition } from '../../generator/helpers'
+import { sharedAddressField } from '../shared/fields'
+
+export default schema(
+  {
+    title: 'TEST - Objednávka vytýčenia',
+  },
+  {},
+  [
+    step('objednavatel', { title: 'Objednávateľ' }, [
+      radioGroup(
+        'objednavatelTyp',
+        {
+          type: 'string',
+          title: 'Objednávate ako',
+          required: true,
+          options: createStringOptions([
+            'Fyzická osoba',
+            'Fyzická osoba - podnikateľ',
+            'Právnická osoba',
+          ]),
+        },
+        { variant: 'boxed', orientations: 'column' },
+      ),
+      conditionalFields(createCondition([[['objednavatelTyp'], { const: 'Fyzická osoba' }]]), [
+        object('menoPriezvisko', { required: true }, { columns: true, columnsRatio: '1/1' }, [
```

Comment:
Unfortunately, you cannot create the same property twice (I need to make this fail when it happens):
```
if fyzicka osoba: meno + priezvisko, **email**, adresa pobytu
if pravnicka osoba: nazov organizacie, **email**, adresa organizacie
```
All of the schemas must be refactored this way (look at OLO forms, it is common there):
```
if fyzicka osoba: meno + priezvisko
if pravnicka osoba: nazov organizacie
email
if fyzicka osoba: adresa pobytu
if pravnicka osoba: adresa organizacie
```

Reply:
This will throw an error when attempting to do this:
https://github.com/bratislava/konto.bratislava.sk/pull/1613